crm_department/__openerp__.py

```diff
@@ -20,22 +20,22 @@
 #
 ##############################################################################
 {
-    'name' : 'CRM with Department Categorization',
-    'version' : '1.0.1',
-    'category' : 'Generic Modules/CRM & SRM',
+    'name': 'CRM with Department Categorization',
+    'version': '1.0.1',
+    'category': 'Generic Modules/CRM & SRM',
     'description': '''
 
 Add the department on Sales Team (with related field on Lead & Opportunities)
 as well as the related filter and button in the search form.
 
```

Comment:
In 8.0 we now move all descriptions in README.rst here is the template to follow:

https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst